ios/StripeSdk.swift

```diff
@@ -602,6 +602,10 @@ class StripeSdk: RCTEventEmitter, STPApplePayContextDelegate, STPBankSelectionVi
             paymentIntentParams.paymentMethodOptions = paymentMethodOptions
             paymentIntentParams.shipping = Mappers.mapToShippingDetails(shippingDetails: params["shippingDetails"] as? NSDictionary)
         }
+        
+        if (paymentMethodType == STPPaymentMethodType.weChatPay) {
+            STPPaymentHandler.shared().simulateAppToAppRedirect = true
```

Comment:
We need to make sure to only set this when when the publishable key includes `test`. Can you add a regex to the if statement to check for that?